includes/discovery/os/nos.inc.php

```diff
@@ -1,7 +1,5 @@
 <?php
 
 if (!$os) {
-    if (strstr($sysObjectId, '.1.3.6.1.4.1.1588.2.2.1.1.1.5') || stristr($sysDescr, 'Brocade Communications Systems')) {
-        $os = 'nos';
-    }
-}
+  if (strstr($sysDescr, "Brocade VDX")) { $os = "nos"; }
```

Comment:
Will this new check pick up the two checks it's removing or should it be in addition to?

Can you separate the { $os = "nos"; } out as before so we can try and keep the code standards up :)